client/my-sites/upgrades/domain-management/style.scss

```diff
@@ -848,20 +848,18 @@ ul.email-forwarding__list {
 	}
 }
 
-.name-servers {
-	.icann-verification {
-		.button {
-			margin-bottom: 15px;
+.icann-verification {
```

Comment:
I wouldn't support `.name-servers` either. This means that this css class will be applied whenever I use `.icann-verification`, it doesn't have to be a component, just any DOM element with that class, wherever it is.